draft-ietf-ohai-ohttp.md

```diff
@@ -1357,17 +1357,29 @@ the chosen key configuration. The Oblivious Relay and Gateway resources can link
 requests using the same key configuration by matching KeyConfig.key\_id, or, if
 the Target Resource is willing to use trial decryption, a limited set of key
 configurations that share an identifier. An Oblivious Relay Resource can link
-requests using the public key corresponding to KeyConfig.key\_id.
-
-Request resources are capable of linking requests depending on how KeyConfigs
-are produced by servers and discovered by clients. Specifically, servers can
-maliciously construct key configurations to track individual clients. A specific
-method for a client to acquire key configurations is not included in this
-specification. Clients need to consider these tracking vectors when choosing a
-discovery method.  Applications using this design should provide accommodations
-to mitigate tracking using key configurations.
-{{?CONSISTENCY=I-D.wood-key-consistency}} provides an analysis of the options
-for ensuring the key configurations are consistent between different clients.
+requests using the public key corresponding to KeyConfig.key\_id or via the
+Oblivious Gateway Resource. We refer to this set of information as the gateway's
+configuration.
+
+Whether or not targets can link requests depends on how gateway configuration
+information is produced and discovered by clients. Specifically, gateways can
+maliciously construct configurations to track individual clients.
+
+Ideally, all clients share a consistent view of the gateway configuration. The number
+of different valid configurations can exponentially partition the anonymity set of
+clients. For example, if there are two valid configurations, that would yield two
+anonymity sets consisting of clients that used one configuration and clients that used
+the other configuration. If the size of these sets are large enough and roughly equal,
+this may not be a total loss of privacy in practice. But it is possible that one
+set is small while the other is large. Configuration consistency is an important
+part of ensuring client anonymity sets are few in number and large in size.
```

Comment:
```suggestion
anonymity sets consisting of clients that used one configuration and clients that used
the other configuration. If the size of each set is large enough, this may not be
a significant loss of privacy in practice. Ensuring that many clients share
a configuration is necessary to provide privacy for clients.
```